### DISCUSSION TITLE
Viewer example

### DISCUSSION THREAD

**Discussion by Ivanderkelen** [CORDEX-be2/ValEnsPy#94] [Ideas]
In the "wild idea" category: 

I came accross this neat example of visualising EURO-CORDEX data.  https://github.com/PBrockmann/EURO-CORDEX_Viewer/tree/master

Might also be something cool to take up with CORDEXBE2, eg hosted on an interactive website? @kobebryant432 @bertvs1 @thomullerBCC 

Might be possible to recycle their code (but need to double check with licenses though).

**Comment by Ivanderkelen** [comment]
the example above seems pretty old (uploaded 10 years ago (!), but here is a more recent example from the Spanish climate scenarios: https://escenarios.adaptecca.es/#&model=EURO-CORDEX-EQM.average&variable=tasmax&scenario=rcp85&temporalFilter=year&layers=AREAS&period=MEDIUM_FUTURE&anomaly=RAW_VALUE 

I guess there are many more to be found!

**Comment by thomullerBCC** [comment]
This is a dashboard that should be relatively easy to develop with Shiny (R or Python), I guess - low cost and efficient because no need for using Javascript, just a bit of HTML and CSS, and possibility to use Leaflet or Mapbox libraries/packages.

**Comment by Ivanderkelen** [comment]
I recently found this climate impacts map from the Netherlands, very elaborate! https://www.klimaateffectatlas.nl/en/
